osu.Game.Tests/Visual/Multiplayer/TestSceneMultiplayerReadyButton.cs

```diff
@@ -43,17 +43,17 @@ private void load(GameHost host, AudioManager audio)
             Dependencies.Cache(rulesets = new RealmRulesetStore(Realm));
             Dependencies.Cache(beatmaps = new BeatmapManager(LocalStorage, Realm, rulesets, null, audio, Resources, host, Beatmap.Default));
             Dependencies.Cache(Realm);
-
-            beatmaps.Import(TestResources.GetQuickTestBeatmapForImport()).WaitSafely();
         }
 
         [SetUp]
         public new void Setup() => Schedule(() =>
         {
             AvailabilityTracker.SelectedItem.BindTo(selectedItem);
 
+            beatmaps.Import(TestResources.GetQuickTestBeatmapForImport()).WaitSafely();
```

Comment:
Can you explain the movement of this line a bit, and what you mean by "ripple through test scene"?